README.md

```diff
@@ -1,2 +1,48 @@
-# rhcos-slb
-Config scripts for RHCOS SLB 
+# Setting up network bonding on specific interfaces during the ignition process
+
+## Overview 
+This guide explains how to bond specific NICs during the initial OpenShift installation with OVS and supplies the Ignition and scripts to achieve this.
+
+## Important
+This guide describes the customisations needed for installation when using a PXE server.
+
+## Prerequisites
+- [Fcct](https://github.com/coreos/butane) software to be able to convert fcc to ign files.
+- PXE server with ability to set kernel arguments.
+- Web accessible location(HTTP only) to upload the custome ign files.
+
+## Usage  
+
+### 1. Adding parameters to PXE server
+Key   |  Type | Value | Exmaple |Discription
+---   | --- | --- | --- | --- |
+ignition.config.url  | String | http://{location reachable to PXE and remote VM in order to get the ignition file}/```file.ign``` custom-config | ignition.config.url=http://my.web.server.redhat.com/```file.ign``` custom-config | This parameter is used to allow the server to load an extra ignition file which captures the MAC addresses and saves them into a file for later use in  machine config.
```

Comment:
Triple quote ``` is not the appropriate symbol here, check how it gets rendered https://github.com/RHsyseng/rhcos-slb/blob/1948132c9d77285f446c48b72af11e66f0d2d64d/README.md

Reply:
Fixed, Added all the HTTP line to the snippet